src/cljs/district_voting/pages/vote_page.cljs

```diff
@@ -4,27 +4,47 @@
     [district-voting.components.contract-info :refer [contract-info]]
     [district-voting.components.how-to-instructions :refer [how-to-instructions]]
     [district-voting.components.voting-bar :refer [voting-bar]]
-    [district-voting.constants :as constants]
     [district-voting.styles :as styles]
     [district0x.components.misc :as misc :refer [row row-with-cols col center-layout paper page]]
-    [re-frame.core :refer [subscribe dispatch]]))
+    [district-voting.proposals.subs :as proposal-subs]
+    [markdown.core :refer [md->html]]
+    [re-frame.core :refer [subscribe dispatch]]
+    [reagent.core :as r]
+    [cljs-react-material-ui.reagent :as ui]
+    [district-voting.components.expandable-text :refer [expandable-text]]
+    )
+  (:require-macros [reagent.ratom :refer [reaction]]))
 
 (defn link [name url]
   [:a {:href url
        :target :_blank}
    name])
 
+(def sort-options {:votes "DNT Votes"
+                   :gh-upvotes "Github upvotes"
+                   :gh-comments "Github comments"
+                   :latest "Latest"})
+
 (defmethod page :route.vote/home []
   (let [votes (subscribe [:voting/candidates-voters-dnt-total :next-district])
         votes-total (subscribe [:voting/voters-dnt-total :next-district])
         loading? (subscribe [:voting-loading? :next-district])
         can-submit? (subscribe [:district0x/can-submit-into-blockchain?])
-        vote-form (subscribe [:form.next-district/vote])]
+        vote-form (subscribe [:form.next-district/vote])
+        proposals (subscribe [::proposal-subs/list :next-district])
+        limit (r/atom 10)
+        sort-order (r/atom :votes)
```

Comment:
I personally would keep limit and sort order in global state rather than ratom. This is not really wrong or anything, just in global state it's a bit better for 2 small reasons: 1. Debugging - when you print state into console, you'll see this. 2. Persistence between figwheel reloads - when you edit this file, you'll lose state in ratoms, which can be pretty annoying is some cases. 
I generally use ratoms only for very internal states of isolated components.